test/test.js

```diff
@@ -3504,6 +3505,48 @@ async function testTournament() {
         });
     });
 
+    await asyncTest("getDivisionInfo()", 2, function() {
+        bc.tournament.getDivisionInfo(
+        _divSetId,
+        function(result) {
+            console.log("heyyy" + result);
+            ok(true, JSON.stringify(result));
+            equal(result.status, 400, "Expecting 200");
+            resolve_test();
+        });
+    });
+
+    await asyncTest("getDivisions()", 0, function() {
```

Comment:
`0` -> `2`
This is the number of sub-tests we expect to pass in this test. There are 2